src/main/java/org/openstreetmap/atlas/geography/atlas/command/AtlasFindByAtlasIdentifierSubCommand.java

```diff
@@ -84,7 +84,7 @@ protected int finish(final CommandMap command)
         // If joining is requested and there are shards to join...
         if (output.isPresent() && !this.shardNames.isEmpty())
         {
-            System.out.printf("Stitching shards and saving to output %s\n", output.get());
+            System.out.printf("Stitching shards and saving to output %s%n", output.get());
```

Comment:
Is this printf intentional?

Reply:
Yes, that one is actually using string formatting.